### PR TITLE
Fixing redirects and removing SNADIs pages

### PR DESCRIPTION
**Pull request by haroldcampos**

### CODE REVIEW

**Comment by haroldcampos** [comment]
@ecfan

File: his/core/application-integration-planning-2.md [hunk 1]
```diff
@@ -20,10 +23,10 @@ Use Visual Studio to work with the TI designer, use configuration files for WIP
 Get details on using single sign-on with Transaction Integrator, limitations, and protecting the different parts and components to limit threats.
 
 ## [Troubleshooting and errors](application-integration-troubleshooting-2.md)
-Read about common issues and resolutions, and using tracing, debugging, and event logs to help troubleshoot. 
+Read about common issues and resolutions, and using tracing, debugging, and event logs to help troubleshoot.
 
 ## Developer resources
 
-[Programmer Guide](application-integration-programmer’s-guide2.md) describes the development tools, using type libraries, and creating or programming Windows-initiated processing (WIP) and Host-initiated processing (HIP) applications. 
+[Programmer Guide](application-integration-programmer’s-guide2.md) describes the development tools, using type libraries, and creating or programming Windows-initiated processing (WIP) and Host-initiated processing (HIP) applications.
```
Comment (ecfan) [inline his/core/application-integration-planning-2.md:30]: Shouldn't the linked file be renamed to fix the apostrophe?

File: his/core/application-integration-planning-2.md [hunk 2]
```diff
@@ -20,10 +23,10 @@ Use Visual Studio to work with the TI designer, use configuration files for WIP
...
 
-[Programmer reference](application-integration-programmer-s-reference2.md) includes tips to handle different data types, working with automated and mainframe data, using COMTIContext, and working with request reply messages. 
+[Programmer reference](application-integration-programmer-s-reference2.md) includes tips to handle different data types, working with automated and mainframe data, using COMTIContext, and working with request reply messages.
```
Comment (ecfan) [inline his/core/application-integration-planning-2.md:32]: The single "s" looks odd in the linked file?

**Review by ecfan — CHANGES_REQUESTED** [review]
Just a couple link issues...

**Comment by haroldcampos** [comment]
Fixed in the last commit @ecfan

**Comment by ecfan** [comment]
@haroldcampos: Please fix these issues:

image

**Review by ecfan — APPROVED** [review]

**Pull request merged**